### PR TITLE
Add unapplied slashes

### PR DESCRIPTION
**Pull request by base1217**
Hello, 

I've added an extra endpoint:
https://polkadot.js.org/docs/substrate/storage#unappliedslasheseraindex-vecunappliedslash

Have already tested it... It works well and very useful for us(at least).

### CODE REVIEW

**Comment by dillu24** [comment]
Hi @base1217 

Thanks for your contribution. The code looks good and everything is in its place, well done!

May I ask you to also do the following please before I merge?:

- In the `doc` folder update the `CHANGELOG.md` file by adding the section pasted below. Please put this section between the `# Change Log` section and the `##1.25.1` section:

```
## Unreleased

* Added the `staking/unappliedSlashes`. This returns all the unapplied slashes in the specified era, or the active era if an era index is not specified.
```
- In the `doc` folder you should also find the `DESIGN_AND_FEATURES.md` file, link: https://github.com/SimplyVC/polkadot_api_server/blob/master/doc/DESIGN_AND_FEATURES.md. Please add the new endpoint to the `Query` section in the table.

After this PR is merged, we will update the `polkadot-js/api` packages to the latest version and issue a new release.

Thanks again!

**Comment by dillu24** [comment]
@DanielMagro97 Any other thing you may wish to add?

**Comment by base1217** [comment]
Hello,
Thanks for feedback.
I added all necessary things. 

Thinking about right way of monitoring validate node or not.

**Comment by dillu24** [comment]
If you require to monitor a validator node, we have PANIC in a separate repo which monitors the nodes. It uses this API server to get the data :)

**Comment by base1217** [comment]
> If you require to monitor a validator node, we have PANIC in a separate repo which monitors the nodes. It uses this API server to get the data :)
Uffff. Looks like huge box which containt many many tools:) We have our own exporter (prometheus, tsdb, allertmanager and so on).
Better for me add this into there. 

The last question:
What's different between /api/query/session/validators and /api/derive/staking/validators (for current session)?

**Comment by dillu24** [comment]
The difference is that `/api/derive/staking/validators` gives you also the set of validators that were elected for the next session.

File: doc/DESIGN_AND_FEATURES.md [hunk 1]
```diff
@@ -61,6 +61,7 @@ The API Server works as follows:
 | `/api/query/staking/erasTotalStake` | `websocket` | `era_index` | The total amount staked in the specified `era index`, or in the `active` one if it is not specified. Value may be in Hex |
 | `/api/query/staking/erasValidatorReward` | `websocket` | `era_index` | The total validator era payout in the specified `era index`, or in the last finished era (active era - 1) if it is not specified |
 | `/api/query/system/events` | `websocket` | `block_hash` | `events` that happened in the specified `block hash`, or in the latest block if the block hash is not specified |
+| `/api/query/staking/unappliedSlashes` | `websocket` | `era_index` | List of `slashed validator`(unnaplied) in the specified era. |
```
Comment (dillu24) [inline doc/DESIGN_AND_FEATURES.md:64]: Can you modify the description to `List of slashed validators (unapplied) in the specified era, or the current era if an era index is not specified.` ?

**Review by dillu24 — CHANGES_REQUESTED** [review]
All seems good. Just a minor addition to the doc and I'll merge.

File: src/interface/substrate_query.js [hunk 2]
```diff
@@ -193,6 +193,28 @@ async function getStakingErasValidatorReward(api, eraIndex) {
     }
 }
 
+async function getUnappliedSlashes(api, eraIndex) {
```
Comment (DanielMagro97) [inline src/interface/substrate_query.js:196]: Can you please rename the function to `getStakingUnappliedSlashes` for consistency with the other functions?

**Review by DanielMagro97 — CHANGES_REQUESTED** [review]

File: src/interface/substrate_query.js (hunk 2 above)
Comment (base1217) [reply src/interface/substrate_query.js]: Hello,

Done:)

Comment (DanielMagro97) [reply src/interface/substrate_query.js]: Thank you!

**Review by dillu24 — APPROVED** [review]
All seems well, gonna merge. Thanks for your contribution!

**Pull request merged**